frontend/public/js/script.js

```diff
@@ -76,10 +76,12 @@ $(document).ready(function () {
 
     $("#picker").change(function(){
         $("#preview-btn").prop("disabled",false);
+	       $("#preview").css("background","");
```

Comment:
It's still not fixed. Make it appear in the same line as the line above and we are good for a merge. Pls use spaces instead of a tab.